README.md

```diff
@@ -1,10 +1,14 @@
 # IREE: Intermediate Representation Execution Environment
 
 IREE (**I**ntermediate **R**epresentation **E**xecution **E**nvironment,
-pronounced as "eerie") is an MLIR-based end-to-end compiler that lowers ML
-models to a unified IR optimized for real-time mobile/edge inference against
-heterogeneous hardware accelerators. IREE also provides flexible deployment
-solutions for the compiled ML models.
+pronounced as "eerie") is an [MLIR](https://mlir.llvm.org/)-based end-to-end
+compiler that lowers Machine Learning (ML) models to a unified IR optimized for
+real-time inference on mobile/edge devices against heterogeneous hardware
+accelerators. IREE also provides flexible deployment solutions for its compiled
+ML models.
+
+See [our website](https://google.github.io/iree/) for project details, user
```

Comment:
I wish this was more prominent, but I can't think of a better way to make that so right now...

Reply:
The website address is also included off to the side: 

![image](https://user-images.githubusercontent.com/4010439/117508777-fbf5fc00-af3d-11eb-830c-7782e1d69a6e.png)